### DISCUSSION TITLE
New data products for TESS-SPOC HLSP at MAST

### DISCUSSION THREAD

**Discussion by scfleming** [lightkurve/lightkurve#1165] [Ideas]
Hi all, just a head's up that TESS-SPOC HLSP at MAST has added data validation products for their light curves, starting now with Sector 36. These are available as optional products in the Observation product list.  These products match those from the mission in terms of file extensions, format, and content. Only those targets with detected TCEs (threshold crossing events) will have DV products available. For more information, consult the TESS-SPOC homepage at MAST (https://archive.stsci.edu/hlsp/tess-spoc). I note this information here in case there are new utility functions (or tutorials) the team would like to create. I also mention it just in case any of the functions were written to assume exactly "N" products with specific file names for TESS-SPOC entries. I would be surprised if that was the case, but if so, there might be some issues or confusion of users if they search for a TESS-SPOC target that has DV files and do not get back only a lc.fits and tp.fits for those.

**Comment by barentsen** [comment]
Thanks for the heads up @scfleming! 

Good to know DV products are now available for TESS-SPOC.  Lightkurve does not currently offer much support for DV products, but I'd encourage anyone to post use cases or ideas for better supporting DV products in this thread.

I'm confident the change will not break existing light curve or target pixel file searches.  Quick sanity check for TOI 201b in Sector 36:

<img width="829" alt="Screen Shot 2021-11-24 at 9 30 16 AM" src="https://user-images.githubusercontent.com/817669/143286986-64f86b30-347e-43f1-b396-b973beb6dd01.png">
